frontend/src/pages/DetailedCourse/DetailedCourseInfo/styled.ts

```diff
@@ -0,0 +1,113 @@
+import { Typography, Button as TextButton } from '@mui/material';
+import styled from 'styled-components';
```

Comment:
Can we change the library? if we planning to remove 'styled-component' in a future

Reply:
As I already mentioned in Overview, in this PR I just reorganised the project structure and didn't changed all the styles or changed using libraries, so it will be done in v1-173 by @ostapdans99 after this PR will be merged